src/openexr.imageio/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ endif()
 
 add_oiio_plugin (${openexr_src}
     INCLUDE_DIRS ${OPENEXR_INCLUDES} ${IMATH_INCLUDE_DIR}/OpenEXR
-    LINK_LIBRARIES ${OPENEXR_LIBRARIES}
```

Comment:
I think this is what's responsible for the break -- we still use OPENEXR_LIBRARIES for very old versions of OpenEXR that we support.

Reply:
Strangely, the "Linux VFX Platform 2019" check also uses OpenEXR 2.3 but it passes.
Thanks for the info, I'll  try to change that part.

Reply:
@lgritz This seems fixed.

The exact issue was the `${OPENEXR_LIBRARIES}` not being included in libOpenImageIO's CMakeLists.txt, which led to the OpenVDB plugin not getting it transitively, which caused problems for older versions of OpenVDB/OpenEXR.

Reply:
In the VFX2019 image, OpenEXR 2.3 is pre-installed in a standard place, and in "oldest" test we build it ourselves in a nonstandard place. So if the former happened to end up in a directory that made it into the `-I` list coincidentally for other packages, then the header would still get found, but only because we got lucky.

Reply:
Thanks for the quick fix, it looks like all tests pass now.